File: matrix/kivalasztas/kivalasztas.cpp
#include <iostream>
#include <vector>

using namespace std;

double Elsoi(vector<double>& lista)
{
    int i = 0;
    while (!(lista[i] < 0))
    {
        i++;
    }
    return i;
}

double Elso1(vector<double>& lista)
{
    return lista[Elsoi(lista)];
}

double Elso2(vector<double>& lista)
{
    for (double& elem : lista)
    {
        if (elem < 0)
        {
            return elem;
        }
    }
}
int main()
{
    vector<double> v = { 4, 5, 1, -2, 3 };
    cout << Elsoi(v) << endl << Elso1(v) << endl << Elso2(v);
}
    

File: matrix/kivalasztas/kivalasztas.py
def elsoi(lista):
    i = 0
    while not lista[i]<0:
        i+=1
    return i

def elso1(lista):
    return lista[elsoi(lista)]

def elso2(lista):
    for elem in lista:
        if elem<0:
            return elem

l = [4, 5, 1, -3, 2]

print(elsoi(l))
print(elso1(l))
print(elso2(l))
